modules/oneVideoBidAdapter.js

```diff
@@ -25,6 +25,17 @@ export const spec = {
       return false;
     }
 
+    // Prevend DAP Outstream validation
+    if (bid.mediaTypes.video) {
+      if (bid.mediaTypes.video.context === 'outstream' && bid.params.video.display === 1) {
+        return false;
+      }
+    }
+    // Banner DAP validation
+    if (bid.mediaTypes.banner && (typeof bid.params.video.display === 'undefined' || !bid.params.video.display)) {
```

Comment:
```
typeof bid.params.video.display === 'undefined' ||
```

^^ This check is redundant, `!bid.params.video.display` would evaluate to `true` in that case as well.

Reply:
understood 👍